cmd/server/server.go

```diff
@@ -42,3 +46,14 @@ func main() {
 
 	server.Listen(host, port)
 }
+
+func setupLogger() {
+	val, ok := os.LookupEnv("LOG_LEVEL")
+	if ok {
+		lvl, err := logrus.ParseLevel(val)
+		if err != nil {
+			return
```

Comment:
Question: is `err` is worth logging here, perhaps at `.Error()`?

Since we used `LookupEnv`, we know the user provided a `LOG_LEVEL` value. They may want feedback if that value was incorrect:
* `err` will give them `not a valid logrus Level: FOO_BAR`
* Overkill, but you could build a message from `logrus.AllLevels` to include what levels are accepted.